scripts/install_third_party.py

```diff
@@ -331,7 +332,9 @@ def download_manifest_files(filepath):
 
             elif download_format == _DOWNLOAD_FORMAT_TAR:
                 dependency_tar_root_name = (
-                    dependency_contents['tarRootDirPrefix'] + dependency_rev)
+                    dependency_contents['tarRootDirPrefix'] +
+                    dependency_rev)
```

Comment:
Is this an expected change?

Reply:
Not sure why this got here. Will revert.

Reply:
Done